src/main/java/com/ncc/aif/AidaDomainOntologiesCommon.java

```diff
@@ -0,0 +1,49 @@
+package com.ncc.aif;
+
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.ResourceFactory;
+
+public final class AidaDomainOntologiesCommon {
+    public static final String NAMESPACE = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/AidaDomainOntologiesCommon#";
+
+    private static AidaDomainOntologiesCommon INSTANCE;
+    private Resource canHaveName = ResourceFactory.createResource(NAMESPACE + "CanHaveName");
+    private Resource canHaveTextValue = ResourceFactory.createResource(NAMESPACE + "CanHaveTextValue");
+    private Resource canHaveNumericValue = ResourceFactory.createResource(NAMESPACE + "CanHaveNumericValue");
+
+    private AidaDomainOntologiesCommon() {
+    }
+
+    public static AidaDomainOntologiesCommon getInstance() {
```

Comment:
I used a singleton in AidaDomainOntologiesCommon because https://fabiomsr.github.io/from-java-to-kotlin/classes.html indicated that was the Java equivalent to the Kotlin "object" declaration.  That said, I have no problem with just making everything accessed statically.